src/main/java/com/intuit/graphql/orchestrator/fieldresolver/FieldResolverBatchSelectionSetSupplier.java

```diff
@@ -153,7 +154,7 @@ private Field createSelectionSetFor(final DataFetchingEnvironment dataFetchingEn
     }
 
     private String compileTemplate(String stringTemplate, Map<String, Object> dataSource) {
-        ValueTemplate valueTemplate = new ValueTemplate(stringTemplate);
+        ValueTemplate valueTemplate = new ValueTemplate(StringUtils.join("#nullCheck(\"",stringTemplate,"\")"));
```

Comment:
I am questioning the use of velocity template, lets discuss more on this during one of our syncups

Reply:
Removed velocity engine and resolved with fieldContext